pkg/testsuite/integration.go

```diff
@@ -222,8 +222,8 @@ func NewIntegrationSuite(tb testing.TB, ctx context.Context) *IntegrationSuite {
 
 	return &IntegrationSuite{
 		cfg:       cfg,
-		db:        db,
-		realm:     realm,
+		DB:        db,
```

Comment:
Why make these public?

Reply:
My test uses the DB and Realm to set up some preconditions for the test.

Specifically it creates an SMSConfig and preloads a VerificationCode to test UUID conflict.